healthcareai/tests/test_trainer.py

```diff
@@ -5,7 +5,7 @@
 from io import StringIO
```

Comment:
2 tests failed - 

AssertionError: 0.90293949226951709 not less than or equal to 0.9

AssertionError: 0.90771139530444744 not less than or equal to 0.9